archlinux/dist

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -o errexit -o pipefail
```

Comment:
Why did you do this instead of `set -e`?

Reply:
Look at the dist from [this repo](https://github.com/Morganamilo/paru/blob/master/dist) it basicly just a copy, just with the changes to build countour, instead of rust code.

Reply:
Changed it.